### DISCUSSION TITLE
Metadata scraping

### DISCUSSION THREAD

**Discussion by TheSinfulKing** [Avnsx/fansly#62] [Ideas]
I'm not familiar with the API, but I was wondering how difficult it would be to add scraping the text associated with posts? I'm thinking the text content of each post could be associated with the output image / video in a SQLite database

**Comment by Avnsx** [comment]
Yes this is definetly possible, but I don't really see the point of adding this. The most users of this scraper, just need the pictures and videos and that completly exceeds the scope of this project to be honest.

**Reply by TheSinfulKing** [reply]
For the archivists like myself lol. 
Having original date is great, so you have some context of the order of downloaded photos and vids.

The original post text would be icing on top
I can write the code myself, what is your dev environment? 

P.S. I use stash app (google it, its on github) and its viewer can import this metadata into a details field, enhancing the viewing experience.
